CIP-0072/version_1_offchain.json

```diff
@@ -11,69 +16,114 @@
-          "enum":["Games", "DeFi", "Gambling", "Exchanges", "Collectibles", "Marketplaces", "Social", "Other"]
+          "enum": ["DeFi", "Development", "Education", "Games", "Identity", "Marketplace", "NFT", "Other", "Security"]
         },
         "description": "One or more categories. Category MUST be one of the following schema definition."
       },
+      "screenshots": {
+        "type": "array",
+        "maxItems": 10,
+        "items": {
+          "description": "Screenshots encoded in base64. Minimum resolution for base64 images: 1920x1080px, supported formats: PNG/JPG/SVG, maximum file size: 2 MB for base64.",
+          "type": "string",
+          "contentEncoding": "base64",
+          "oneOf": [
+            {
+              "contentMediaType": "image/png"
+            },
+            {
+              "contentMediaType": "image/jpeg"
+            },
+            {
+              "contentMediaType": "image/svg+xml"
+            }
+          ],
+          "maxLength": 2722000
+        },
+        "description": "Screenshots of the DApp encoded in base64. We recommend to share screenshots from the dApp usage itself."
+      },
       "social": {
-        "type":"object",
-        "properties": {
-          "website": {
-            "type":"string",
-            "description": "dApps website link.",
-            "pattern": "(https?:\/\/(?:www\\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\\.[^\\s]{2,}|www\\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\\.[^\\s]{2,}|https?:\/\/(?:www\\.|(?!www))[a-zA-Z0-9]+\\.[^\\s]{2,}|www\\.[a-zA-Z0-9]+\\.[^\\s]{2,})"
-          },
-          "twitter": {
-            "type":"string",
-            "description": "An optional Twitter link.",
-            "pattern": "(https?:\/\/(?:www\\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\\.[^\\s]{2,}|www\\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\\.[^\\s]{2,}|https?:\/\/(?:www\\.|(?!www))[a-zA-Z0-9]+\\.[^\\s]{2,}|www\\.[a-zA-Z0-9]+\\.[^\\s]{2,})"
-          },
-          "github": {
-            "type":"string",
-            "description": "An optional Github link.",
-            "pattern": "(https?:\/\/(?:www\\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\\.[^\\s]{2,}|www\\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\\.[^\\s]{2,}|https?:\/\/(?:www\\.|(?!www))[a-zA-Z0-9]+\\.[^\\s]{2,}|www\\.[a-zA-Z0-9]+\\.[^\\s]{2,})"
-          },
-          "discord": {
-            "type":"string",
-            "description": "An optional Discord link.",
-            "pattern": "(https?:\/\/(?:www\\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\\.[^\\s]{2,}|www\\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\\.[^\\s]{2,}|https?:\/\/(?:www\\.|(?!www))[a-zA-Z0-9]+\\.[^\\s]{2,}|www\\.[a-zA-Z0-9]+\\.[^\\s]{2,})"
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string",
+              "description": "The platform or resource identifier (GitHub, Website, X.com, etc)"
+            },
+            "link": {
```

Comment:
All the other links have max 200 chars allowed. Should we have it here too?

Reply:
good point, thank you 😉